src/common/crc32c_ppc.c

```diff
@@ -35,8 +37,58 @@ static unsigned int crc32_align(unsigned int crc, unsigned char const *p,
 }
 #endif
 
-
 #ifdef HAVE_POWER8
+static inline unsigned long polynomial_multiply(unsigned int a, unsigned int b) {
+        vector unsigned int va = {a, 0, 0, 0};
+        vector unsigned int vb = {b, 0, 0, 0};
+        vector unsigned long vt;
+
+        __asm__("vpmsumw %0,%1,%2" : "=v"(vt) : "v"(va), "v"(vb));
+
+        return vt[0];
+}
+
+static unsigned int reflect32(unsigned int v) {
```

Comment:
Same function is defined in src/common/hobject.h. I suggest that you move it from there into some common module and use that instead of copying it here.